code.go

```diff
@@ -13,12 +13,15 @@ func (c code) Error() string {
 
 	if bytes.Equal(c, []byte{0x6a, 0x80}) {
```

Comment:
Here you did not introduce constants? Could you also include the constant documentation links while at it (in `constants.go`?)?